src/style.css

```
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars-block {
+  flex-shrink: 0;
+  margin: 0;
+  padding: 0;
+}
+
+.stars{
+  display: flex;
+  margin: 0;
+}
+
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars--1 :first-child {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
```

you can marge like this 
```suggestion

.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.... {
  background-image: url(images/star-active.svg);
}
```